src/server/ApiRoutes/Bib.js

```diff
@@ -141,6 +141,11 @@ function fetchBib(bibId, cb, errorcb, reqOptions) {
 
       return data;
     })
+    .then((bib) => {
+      const status = (!bib || !bib.uri || bib.uri !== bibId) ? '404' : '200';
+      if (status === '404') res.nyplStatus = 404;
+      return Object.assign({ status }, bib);
```

Comment:
I think it makes sense to have it on the bib, since it is really the status of the request for that particular bib. I could also see putting it in the store but my worry there would be that an old `status` for one request gets incorrectly applied later if it isn't cleared for some reason. Putting `status` on the bib avoids that issue.